pm/sender_test.go

```diff
@@ -90,6 +90,49 @@ func TestSenderEV(t *testing.T) {
 	assert.Zero(ticketEV(ticketParams.FaceValue, ticketParams.WinProb).Cmp(ev))
 }
 
+func TestValidateSender(t *testing.T) {
+	assert := assert.New(t)
+	account := accounts.Account{
+		Address: RandAddress(),
+	}
+	am := &stubSigner{
+		account: account,
+	}
+	rm := &stubRoundsManager{round: big.NewInt(5), blkHash: [32]byte{5}}
+	sm := newStubSenderManager()
+	sm.info[account.Address] = &SenderInfo{
+		Deposit:       big.NewInt(100000),
+		WithdrawRound: big.NewInt(0),
+	}
+	s := &sender{
+		signer:            am,
+		roundsManager:     rm,
+		senderManager:     sm,
+		maxEV:             big.NewRat(100, 1),
+		depositMultiplier: 2,
+	}
+	// GetSenderInfo error
+	sm.err = errors.New("GetSenderInfo error")
+	err := s.validateSender()
+	assert.EqualError(err, "unable to validate sender: could not get sender info: GetSenderInfo error")
+	sm.err = nil
+
+	// sender's (withdraw round + 1 <= current round)
+	sm.info[account.Address].WithdrawRound = big.NewInt(4)
+	err = s.validateSender()
+	assert.EqualError(err, "unable to validate sender: deposit and reserve is set to unlock soon")
+	sm.info[account.Address].WithdrawRound = big.NewInt(0)
+
+	// Sufficient deposit and not unlocked
```

Comment:
We aren't checking the deposit any more so this and the other comment below need to be updated

Reply:
fixed in 4fb8b5d2